modules/configuring-pipelines-with-your-own-argo-workflows-instance.adoc

```diff
@@ -0,0 +1,40 @@
+:_module-type: PROCEDURE
+
+[id="configuring-pipelines-with-your-own-argo-workflows-instance_{context}"]
+= Configuring pipelines with your own Argo Workflows instance
+
+[role="_abstract"]
+You can use your own Argo Workflows instance with {productname-short} pipelines. This configuration disables the embedded Argo Workflows controllers that are managed by the {productname-short} Operator and enables your external Argo Workflows instance.
+
+You cannot enable both the embedded Argo Workflows instance and your own Argo Workflows instance on the same cluster.
+
+.Prerequisites
+* You have cluster administrator privileges for your {openshift-platform} cluster.
+* You have installed {productname-long}.
+
+.Procedure
+. Log in to the {openshift-platform} web console as a cluster administrator.
+. In the {openshift-platform} console, click *Operators* → *Installed Operators*.
+. Search for the *{productname-long}* Operator, and then click the Operator name to open the Operator details page.
+. Click the *Data Science Cluster* tab.
+. Click the default instance name (for example, *default-dsc*) to open the instance details page.
+. Click the *YAML* tab to show the instance specifications.
+. Disable the embedded Argo Workflows controllers that are managed by the {productname-short} Operator:
+.. In the `spec.components` section, set the `managementState` field for the `datasciencepipelines` component to `Managed`.
+.. In the `spec.components.datasciencepipelines` section, set the value of the `argoWorkflowControllers.managementState` field to `Removed`, as shown:
++
```

Comment:
```suggestion
+.Sample datasciencepipelines specification
```

Reply:
Suggestion to add a little more context to clarify that it is a sample yaml. also suggest including some #... before and after the spec to show that content precedes and continues